amqp-impl/src/main/java/io/streamnative/pulsar/handlers/amqp/impl/PersistentQueue.java

```diff
@@ -103,6 +108,38 @@ public Topic getTopic() {
         return indexTopic;
     }
 
+    public void recoverRoutersFromQueueProperties(Map<String, String> properties, AmqpBrokerService amqpBrokerService,
```

Comment:
Can we just pass in the `ExchangeContainer` and the `PulsarService`, instead of the whole `AmqpBrokerService`?  Those are the only 2 objects that are being used, and it will make the function's dependencies more explicit.

Reply:
OK, seems good.